src/fhir/guides/ig-atf/Home/Einfuehrung.page.md

```diff
@@ -1,41 +1,76 @@
+
+- **Sicherheit und Effizienz**: Die Einführung einer automatisierten **Empfangsbestätigung** gewährleistet, dass jede Nachricht korrekt empfangen und verarbeitet wurde. Der Absender erhält Rückmeldung über das Ergebnis der Verarbeitung, was die Effizienz des Kommunikationsprozesses verbessert und Fehler schnell identifiziert.
+
+- **Fehlertoleranz**: Durch das **Fallback-Szenario mit XSLT-Stylesheets** können Nachrichten bei Bedarf als menschenlesbare PDF-Dokumente übermittelt werden, falls technische Probleme die maschinelle Verarbeitung verhindern. Dies erhöht die Ausfallsicherheit und stellt sicher, dass der Inhalt der Nachricht dennoch zugänglich bleibt.
+
+**Zusammenfassung**: Stufe 1 legt die Grundlage für eine strukturierte, transparente und fehlertolerante Kommunikation im medizinischen Umfeld. Standardisierte Use Cases, klare Benennung der Systeme, automatisierte Empfangsbestätigungen und Fallback-Mechanismen verbessern die Sicherheit und Effizienz des Nachrichtenaustauschs erheblich.
+
+### Outlook: Automatisierter Use Case-Abgleich
+
+Als nächster Schritt wird das App-Transport-Framework um **Capability-Nachrichten** erweitert. Diese Nachrichten ermöglichen eine **machine-to-machine-Aushandlung** zwischen Sender und Empfänger, um zu verhandeln, welche Use Cases sie unterstützen und wie die Kommunikation ablaufen soll. Diese Erweiterung wird jedoch nur spezifiziert, wenn aus der Industrie oder von Anwendern eine klare Notwendigkeit dafür signalisiert wird. Das Framework bleibt somit anpassungsfähig und wird entsprechend den Anforderungen der Praxis erweitert.
 
 ## Implementierung des Proof of Concept
 
-Die exemplarische [Implementierung des App-Transport-Frameworks](https://github.com/gematik/api-app-transport-framework/blob/main/src/poc/README.md) und einer [Beispielbibliothek](https://github.com/gematik/api-app-transport-framework/blob/main/src/app_transport_framework_library/README.md) demonstriert die Anwendung in Python.
+Ein [Proof of Concept](https://github.com/gematik/api-app-transport-framework/blob/main/src/poc/README.md) sowie eine [Beispielbibliothek](https://github.com/gematik/api-app-transport-framework/blob/main/src/app_transport_framework_library/README.md) demonstrieren die praktische Anwendung des App-Transport-Frameworks in Python.
 
-Eine exemplarische Verwendung dieser Bibliothek, in der Form der Übermittlung einer Selbsttest-Nachricht, ist in einem [Beispiel](https://github.com/gematik/api-app-transport-framework/blob/main/src/poc/example.py) dargestellt.
+Ein [Beispiel-Szenario](https://github.com/gematik/api-app-transport-framework/blob/main/src/poc/example.py), das die Übermittlung einer Selbsttest-Nachricht zeigt, verdeutlicht die Verwendung der Bibliothek in der Praxis.
```

Comment:
Selbsttest wurde noch nicht definiert, bzw. ein Link gegeben wo ich dazu was lesen kann wie das funktioniert

Reply:
Das stimmt. Aber es wird auf der Seite nicht ein mal erwähnt. Als Leser weiß ich nicht wofür das ist, was ich damit erreiche etc.
Kann sein, dass das ein in der Entwicklung bekanntes Prinzip ist, was ich nicht kenne, dann ist es wahrscheinlich ok.